signs-work-order-map/utils/mapUtils.tsx

```diff
@@ -91,10 +95,39 @@ export const useFormatLocation = (
     };
   }, [knackPayload]);
 
+/**
+ * Custom marker component for AGOL signs (yellow dots with black stroke)
+ */
+const AGOLMarker = ({ onClick }: { onClick: (e: any) => void }) => (
+  <div
+    onClick={onClick}
+    style={{
+      width: "8px",
+      height: "8px",
+      borderRadius: "50%",
+      backgroundColor: "#FFD700", // Gold/Yellow color
+      border: "1px solid #000000", // Black stroke
+      cursor: "pointer",
+      boxShadow: "0 1px 2px rgba(0,0,0,0.3)", // Subtle shadow for visibility
+      transform: "translate(-50%, -50%)", // Center the dot on the coordinate
+    }}
+  />
+);
+
+/**
+ * Helper function to log AGOL sign data to console
+ */
+const logAGOLSignData = (sign: Sign) => {
```

Comment:
This could be deleted once we implement interactivity with each sign asset point. For now, I just wanted to demonstrate high-level details about each point for future iteration.

Reply:
Or, maybe leave it in and make it use `console.info()` - my motivation for this comment is that the console logging you've got going here is really good in how it conveys events, counts etc..  and is just generally an example of information-dense logging that is also readable at the same time. I'd hate to loose it because i think future us would love to have it stick around when we pick it back up to dev on.